File: Programming Basics with Python/03 Conditional statements advanced/More exercises/07_School_Camp.py
season = input().lower()
group_type = input().lower()
count_students = int(input())
nights = int(input())
price_per_night = 0
sport = ""
if season == "winter":
    price_per_night = 9.6
    if group_type == "mixed":
        price_per_night = 10
        sport = "Ski"
    elif group_type == "girls":
        sport = "Gymnastics"
    else:
        sport = "Judo"
elif season == "spring":
    price_per_night = 7.2
    if group_type == "mixed":
        price_per_night = 9.5
        sport = "Cycling"
    elif group_type == "girls":
        sport = "Athletics"
    else:
        sport = "Tennis"
elif season == "summer":
    price_per_night = 15
    if group_type == "mixed":
        price_per_night = 20
        sport = "Swimming"
    elif group_type == "girls":
        sport = "Volleyball"
    else:
        sport = "Football"
costs = count_students * price_per_night *nights
if count_students >= 50:
    costs *= 0.5
elif 20 <= count_students < 50:
    costs *= 0.85
elif 10 <= count_students < 20:
    costs *= 0.95
print(f"{sport} {costs:.2f} lv.")

File: JS Basics/03 Nested Conditional Statements - More Exercises/07. School Camp.js
function camp(season, group, students, nights) {
    students = Number(students);
    nights = Number(nights);
    let price = students * nights;
    let sport = "";
    if (season == "Winter") {
        if (group == "boys") {
            price *= 9.60;
            sport = "Judo";
        }
        else if (group == "girls") {
            price *= 9.60;
            sport = "Gymnastics";
        }
        else {
            price *= 10;
            sport = "Ski";
        }
    }
    else if (season == "Summer") {
        if (group == "boys") {
            price *= 15;
            sport = "Football";
        }
        else if (group == "girls") {
            price *= 15;
            sport = "Volleyball";
        }
        else {
            price *= 20;
            sport = "Swimming";
        }
    }
    else {
        if (group == "boys") {
            price *= 7.2;
            sport = "Tennis";
        }
        else if (group == "girls") {
            price *= 7.2;
            sport = "Athletics";
        }
        else {
            price *= 9.5;
            sport = "Cycling";
        }
    }
    if (students >= 50) {
        price *= 0.5;
    }
    else if (students >= 20) {
        price *= 0.85;
    }
    else if (students >= 10) {
        price *= 0.95;
    }
    console.log(`${sport} ${price.toFixed(2)} lv.`);
}
camp("Spring", "mixed", 17, 14)